server/src/handlers/message_handler.rs

```diff
@@ -132,7 +133,7 @@ pub struct CreateMessageReqPayload {
         ("TR-Dataset" = uuid::Uuid, Header, description = "The dataset id or tracking_id to use for the request. We assume you intend to use an id if the value is a valid uuid."),
     ),
     security(
-        ("ApiKey" = ["readonly"]),
+        ("ApiKey" = ["admin"]),
```

Comment:
Is this intential? upping the permissions needed to create a message could break UI's

Reply:
It's just for the docs. We have been requiring admin here for a long time, the openapi spec was just off.